inst/include/dust/xoshiro.hpp

```diff
@@ -51,15 +53,15 @@ inline uint64_t xoshiro_next(rng_state_t<T>& state) {
   return result;
 }
 
-inline uint64_t splitmix64(uint64_t seed) {
+inline HOST uint64_t splitmix64(uint64_t seed) {
```

Comment:
Just in case this wasn't clear, we don't really need to use `HOST`, but I put them explicitly in all the functions in this file as the RNG was trickiest to get right on both CPU and GPU, and this seemed to make it clear which you are allowed to use in kernels